### ISSUE TITLE
Update locales from closure-library

### ISSUE THREAD

**Issue by rafasoares** [angular/angular.js#15976]
As instructed on https://github.com/angular/angular.js/pull/11925#issuecomment-116909287, I'm opening a new issue now that https://github.com/google/closure-library/blob/master/closure/goog/i18n/datetimesymbols.js is updated with the CLDR release 31 data (which includes the changes in the mentioned PR) and Angular needs to be updated accordingly.

I just checked the code on the master branch and the v1.6.x branch, and both seem outdated.

Thank you.

**Comment by rafasoares** [comment]
@gkalpak care to explain what does the `severity: confusing` label means? Do you need me to explain anything? I'll be happy to, if that's the case.

**Comment by gkalpak** [comment]
According to [TRIAGING.md][1] it means _"unexpected or inconsistent behavior; hard-to-debug"_. This is not exactly what I meant, but it was the closest. It refers to the "thing" you are reporting, not the report (GitHub issue) itself. No need to explain anything :smiley:

I have put it on the 1.6.5 milestone, so it should be done before the 1.6.5 release. And right now might as good a time as any :smiley:

[1]: https://github.com/angular/angular.js/blob/master/TRIAGING.md
